phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1788,6 +1799,50 @@ private MetaDataMutationResult childLinkMetaDataCoprocessorExec(byte[] parentTab
         }
     }
 
+    @VisibleForTesting
+    protected RpcController getController() {
+        return getController(SYSTEM_CATALOG_HBASE_TABLE_NAME);
+    }
+
+        /**
+         * If configured to use the metadata handler pool for server side connections,
+         * use the {@link org.apache.hadoop.hbase.ipc.controller.MetadataRpcController}
```

Comment:
This comment appears to be incorrect (it says it's using the MetadataRpcController)